delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ElemRoutes.scala

```diff
@@ -57,18 +57,18 @@ class ElemRoutes(
                 concat(
                   (get & pathPrefix("continuous") & parameter("tag".as[UserTag].?)) { tag =>
```

Comment:
We need to get the types as an input there to pass them down as an input.
So also a change in the `DeltaClient`